src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double PI = 3.14;
```

Comment:
remove this variable, it shouldn't be here, if you need this value, use it directly in the method where you need it

Reply:
[constant removed](https://github.com/Maksym-Kulakov/jv-oop-advanced/commit/5b427ab324329e82425a01bcbcbdc8c89a044484)